lib/keystore.hh

```diff
@@ -35,7 +36,8 @@ public:
     rpmRC delete_key(rpmtxn txn, rpmPubkey key) override;
     rpmRC delete_store(rpmtxn txn) override;
 private:
-    rpmRC delete_key(rpmtxn txn, const std::string & keyid, unsigned int newinstance = 0);
+    friend rpmRC delete_key(auto keystore, rpmtxn txn, rpmPubkey key, auto skip);
```

Comment:
Lets group these friend declarations separate from the actual methods for clarity.

Reply:
Oh right, they're not "private" methods to start with, either. Will do.